src/app/content/highlights/components/Card.tsx

```diff
@@ -120,7 +124,15 @@ const Card = (props: CardProps) => {
     className: props.className,
     highlight: props.highlight,
     isFocused: props.isFocused,
-    onBlur: props.blur,
+    onBlur: () => {
+      if (props.scrollTarget) {
+        services.history.replace({
```

Comment:
again, i don't think we have to clear the scroll target until they set a new one or navigate to a new page (when it should be cleared on its own)